src/components/auth/AuthHeader.tsx

```diff
@@ -17,18 +17,18 @@ const AuthHeader = () => {
       {!isLoggedIn ? (
         <div className="flex items-center gap-3 sm:gap-3">
           <Link href="/login">
-            <Button className="inline-flex items-center justify-center gap-1.5 rounded-lg border border-gray-200 px-5 py-3 text-gray-500 transition hover:bg-gray-50 hover:text-gray-700 focus:outline-none focus:ring">
+            <Button className="bg-primary-400 hover:bg-primary dark:bg-primary-400 dark:text-white dark:hover:bg-primary">
               Login
             </Button>
           </Link>
-          <Link href="/register">
+          {/* <Link href="/register">
             <Button
               variant="secondary"
```

Comment:
better to delete the commented code, unless it's saved for later

Reply:
btw, why is it commented?

Reply:
It's because there is no signup button in new design, and I thought we might need that button in future after discussion, that's why I commented it out rather than deleting.